src/utils/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum Error {
```

Comment:
`Error` name for will conflict with global Error object so you can have some unpredictable behaviour. Better rename it.